app/helpers/injection_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module InjectionHelper
   def inject_enterprises
-    inject_json_ams "enterprises", Enterprise.activated.includes(:address).all, Api::EnterpriseSerializer, enterprise_injection_data
+    inject_json_ams "enterprises", Enterprise.activated.includes(address: :state).all, Api::EnterpriseSerializer, enterprise_injection_data
```

Comment:
Thanks. That's great to know. And a really good improvement. We have a lot of queries...